### ISSUE TITLE
Error installing in node / mac

### ISSUE THREAD

**Issue by lmammino** [capnproto/node-capnp#37]
## OS info:

```
System Version: macOS 10.13.2 (17C205)
Kernel Version: Darwin 17.3.0
node -v: v9.2.1
npm -v: 5.6.0
```

## Error report:

```
npm i --save capnp

> capnp@0.2.0 install project/node_modules/capnp
> node ./build.js

(node:4319) [DEP0006] DeprecationWarning: child_process: options.customFds option is deprecated. Use options.stdio instead.

  CXX(target) Release/obj.target/capnp/src/node-capnp/capnp.o
../src/node-capnp/capnp.cc:885:43: error: no member named 'GetHiddenValue' in 'v8::Object'
      v8::Handle<v8::Value> native = obj->GetHiddenValue(newSymbol("capnp::native"));
                                     ~~~  ^
../src/node-capnp/capnp.cc:1006:33: error: no member named 'SetHiddenValue' in 'v8::Object'
    v8::Object::Cast(*args[0])->SetHiddenValue(newSymbol("capnp::native"), args[1]);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
../src/node-capnp/capnp.cc:1433:40: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
          auto wrapped = localCapType->NewInstance(1, &arg);
                                       ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3843:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:1451:42: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
            auto wrapped = localCapType->NewInstance(1, &arg);
                                         ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3843:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:1680:24: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
        result = func->NewInstance(kj::size(args), args);
                       ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3843:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:1696:26: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
          result = func->NewInstance(kj::size(args), args);
                         ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3843:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:1869:14: error: no member named 'SetHiddenValue' in 'v8::Object'
    wrapper->SetHiddenValue(newSymbol("buffer"), bufferHandle);
    ~~~~~~~  ^
../src/node-capnp/capnp.cc:1933:22: warning: 'restore' is deprecated: Please transition to using a bootstrap
      interface instead. [-Wdeprecated-declarations]
    return rpcSystem.restore(hostId, objectId);
                     ^
/usr/local/include/capnp/rpc.h:102:7: note: 'restore' has been explicitly marked deprecated here
      KJ_DEPRECATED("Please transition to using a bootstrap interface instead.");
      ^
/usr/local/include/kj/common.h:192:20: note: expanded from macro 'KJ_DEPRECATED'
    __attribute__((deprecated(reason)))
                   ^
../src/node-capnp/capnp.cc:2168:34: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
              fieldValue = func->NewInstance(kj::size(args), args);
                                 ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3843:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version",
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:1020:35: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<capnp::Schema>' requested here
    auto result = context.wrapper.wrap(new capnp::Schema(schema));
                                  ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<capnp::InterfaceSchema::Method>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:1091:38: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<capnp::InterfaceSchema::Method &>' requested here
                     context.wrapper.wrapCopy(method));
                                     ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:1253:28: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::StructBuilder>' requested
      here
    return context.wrapper.wrap(new StructBuilder(schema.asStruct()));
                           ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<capnp::DynamicCapability::Client>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:1676:54: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<capnp::DynamicCapability::Client>' requested here
      v8::Handle<v8::Value> result = context.wrapper.wrapCopy(kj::mv(cap));
                                                     ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:1842:35: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::FlatStructReader>' requested
      here
        wrapper = context.wrapper.wrap(new FlatStructReader(words, schema.asStruct()));
                                  ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<kj::Array<capnp::word> >' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:1843:40: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<kj::Array<capnp::word> >' requested here
        bufferHandle = context.wrapper.wrapCopy(kj::mv(words));
                                       ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:1845:35: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::PackedStructReader>'
      requested here
        wrapper = context.wrapper.wrap(new PackedStructReader(
                                  ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:1865:35: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::StructReader>' requested here
        wrapper = context.wrapper.wrap(new StructReader(words, schema.asStruct()));
                                  ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::ConnectionWrapper>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:1984:28: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<v8capnp::(anonymous namespace)::ConnectionWrapper>'
      requested here
    return context.wrapper.wrapCopy(ConnectionWrapper { promise.fork() });
                           ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::ClientRequest>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2128:28: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<v8capnp::(anonymous namespace)::ClientRequest>' requested
      here
    return context.wrapper.wrapCopy(ClientRequest { cap.newRequest(method) });
                           ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<kj::Own<v8capnp::(anonymous namespace)::Canceler> >' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2227:53: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<kj::Own<v8capnp::(anonymous namespace)::Canceler> >'
      requested here
    v8::Handle<v8::Object> result = context.wrapper.wrapCopy(kj::addRef(*canceler));
                                                    ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::ClientResponse>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2238:25: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<v8capnp::(anonymous namespace)::ClientResponse>'
      requested here
        context.wrapper.wrapCopy(ClientResponse { kj::mv(response) })
                        ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::ServerRequest>' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2349:54: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<v8capnp::(anonymous namespace)::ServerRequest>' requested
      here
    v8::Handle<v8::Value> arg = capnpContext.wrapper.wrapCopy(kj::mv(request));
                                                     ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<kj::Own<kj::PromiseFulfiller<capnp::Capability::Client> > >' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2412:25: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<kj::Own<kj::PromiseFulfiller<capnp::Capability::Client> >
      >' requested here
        context.wrapper.wrapCopy(kj::mv(paf.fulfiller)));
                        ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:874:12: note: in instantiation of function template specialization 'v8capnp::(anonymous
      namespace)::Wrapper::wrap<kj::Own<v8capnp::(anonymous namespace)::ServerResults> >' requested here
    return wrap(new kj::Decay<T>(kj::fwd<T>(value)));
           ^
../src/node-capnp/capnp.cc:2498:28: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrapCopy<kj::Own<v8capnp::(anonymous namespace)::ServerResults> >'
      requested here
    return context.wrapper.wrapCopy(kj::mv(results));
                           ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
../src/node-capnp/capnp.cc:867:74: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
    v8::Local<v8::Object> obj = getFunctionTemplate<T>()->GetFunction()->NewInstance();
                                                                         ^
../src/node-capnp/capnp.cc:2738:42: note: in instantiation of function template specialization
      'v8capnp::(anonymous namespace)::Wrapper::wrap<v8capnp::(anonymous namespace)::CapnpContext>' requested here
  auto wrappedContext = context->wrapper.wrap(context);
                                         ^
$HOME/.node-gyp/9.2.1/include/node/v8.h:3848:3: note: 'NewInstance' has been explicitly marked
      deprecated here
  V8_DEPRECATED("Use maybe version", Local<Object> NewInstance() const);
  ^
$HOME/.node-gyp/9.2.1/include/node/v8config.h:321:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
22 warnings and 3 errors generated.
make: *** [Release/obj.target/capnp/src/node-capnp/capnp.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit ($HOME/n/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:159:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:12)
gyp ERR! System Darwin 17.3.0
gyp ERR! command "$HOME/n/bin/node" "$HOME/n/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd project/node_modules/capnp
gyp ERR! node -v v9.2.1
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok
Build failed
npm WARN lastfm-to-elastic@1.0.0 No description
npm WARN lastfm-to-elastic@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! capnp@0.2.0 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the capnp@0.2.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     $HOME/.npm/_logs/2018-01-12T21_43_49_545Z-debug.log
```

**Comment by lmammino** [comment]
Same seems to be happening with node `v8.9.4` and `v7.10.1`, but it does seem to work in `v6.12.3`

**Comment by kentonv** [comment]
Have you tried building from source using the `node8` branch? I think that should work. I haven't been very good about publishing changes to npm. :/

**Comment by sachnk** [comment]
I run into the same issue on Mac and Linux, but building from source seems to work. Any chance the npm package could be updated for easier dependency management?

**Comment by kentonv** [comment]
I just published to npm version 0.3.0 which targets Node 8+. Please let me know if you have any problems with it. Sorry for the delay.
